backend/app/controllers/appController.js

```diff
@@ -1,46 +1,46 @@
-const { getRestaurantFromId, getRestaurantFromName, getRestaurantFromHostname } = require('../datamodels');
+const { getRestaurantById, getRestaurantByName, getRestaurantByHostname } = require('../datamodels');
 
-const getRestaurantById = async (req, res) => {
-  const { params: { restaurantId } } = req;
-  console.log(req.params);
+const getRestaurantByIdController = async (req, res) => {
+	const { params: { restaurantId } } = req;
+	console.log(req.params);
 
-    try {
-      const result = await getRestaurantFromId(restaurantId);
-      res.send(result);
-    } catch (error) {
-      res.sendStatus(404);
-    }
+	try {
+		const result = await getRestaurantById(restaurantId);
+		res.send(result);
+	} catch (error) {
+		res.sendStatus(404);
+	}
 };
 
-const getRestaurantByName = async (req, res) => {
-  const { params: { restaurantName } } = req;
+const getRestaurantByNameController = async (req, res) => {
+	const { params: { restaurantName } } = req;
 
-    try {
-      const result = await getRestaurantFromName(restaurantName);
-      res.send(result);
-    } catch (error) {
-      res.sendStatus(404);
-    }
+	try {
+		const result = await getRestaurantByName(restaurantName);
+		res.send(result);
+	} catch (error) {
+		res.sendStatus(404);
+	}
 };
 
 
-const hostnameCheck = async (req, res) => {
-  const { body: { hostname } } = req;
-  if (!hostname) {
-    res.sendStatus(404);
-  } else {
-    try {
-      const result = await getRestaurantFromHostname(hostname);
-      if(!result) res.sendStatus(404);
-      res.send(result);
-    } catch (error) {
-      res.sendStatus(404);
-    }
-  }
+const getRestaurantByHostnameController = async (req, res) => {
+	const { body: { hostname } } = req;
+	if (!hostname) {
+		res.sendStatus(404);
+	} else {
+		try {
+			const result = await getRestaurantByHostname(hostname);
+			if (!result) res.sendStatus(404);
+			res.send(result);
+		} catch (error) {
+			res.sendStatus(404);
+		}
+	}
 };
 
 module.exports = {
-  getRestaurantById,
-  getRestaurantByName,
-  hostnameCheck, 
+	getRestaurantByIdController,
+	getRestaurantByNameController,
+	getRestaurantByHostnameController,
 };
```

Comment:
The code patch seems to have changed the names of three functions for querying a restaurant in a data model and turned them into controller functions that can handle incoming requests from users. The changes mainly involve renaming the original function names to fit with the new naming conventions. Additional changes involve using more expressive, descriptive variable names for better readability. 

One minor issue is that in the `hostnameCheck` function, if `result` is undefined, it sends back 404, which implies that the resource isn't found, which could be misleading. A better approach might be sending back 204 (no content) instead.

Overall, the code patch looks appropriate as long as there are no issues with the data models and the request handling of the endpoints works as intended.